app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -302,12 +302,8 @@ function handleFocus(event) {
   var area = event.target;
   if (userStatus == 'logged-out') {
     event.preventDefault();
-    showModal('reply-to-comment');
+    showLoginModal('reply-to-comment');
     area.blur();
-    setTimeout(function () {
-      area.blur();
-      showModal('reply-to-comment');
-    }, 100);
```

Comment:
This was causing the modal to keep popping up. I've added a new issue to look at this behaviour in more detail as I think it needs some rework: #12662